extensions/amp-timeago/validator-amp-timeago.protoascii

```diff
@@ -44,6 +44,8 @@ tags: {  # <amp-timeago>
     value_regex: "\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d(:[0-5]\\d(\\.\\d+)?)?(Z|[+-][0-1][0-9]:[0-5][0-9])"
   }
   attrs: { name: "locale" }
+  # <amp-bind>
+  attrs: { name: "[datetime]" }
```

Comment:
worth updating validator_time-ago.html and .out files as well

Reply:
I've added some new validator test files for mutations - is this OK?

Reply:
That's fine.